sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -3,18 +3,107 @@ import { display } from "@microsoft/fast-foundation";
 
 export const ContentPlacementContainerStyles = css`
     ${display("block")}:host {
+        --border: calc(var(--outline-width) * 0.5px) solid var(--neutral-outline-rest);
         contain: none;
         font-family: var(--body-font);
     }
-    .container {
-        display: flex;
-        flex-wrap: wrap;
+
+    .framework_container,
+    .community_container {
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
         justify-content: center;
     }
+
+    .framework_ContentPlacement {
+        padding: 0 calc(var(--design-unit) * 2px);
+    }
+
+    .framework_ContentPlacement:nth-of-type(4),
+    .framework_ContentPlacement:nth-of-type(5),
+    .framework_ContentPlacement:nth-of-type(6) {
+        border-top: var(--border);
+    }
+
+    .framework_ContentPlacement:nth-of-type(1),
+    .framework_ContentPlacement:nth-of-type(2),
+    .framework_ContentPlacement:nth-of-type(4),
+    .framework_ContentPlacement:nth-of-type(5) {
+        border-right: var(--border);
+    }
+
+    .framework_ContentPlacement:nth-of-type(1),
+    .framework_ContentPlacement:nth-of-type(2),
+    .framework_ContentPlacement:nth-of-type(3) {
+        border-bottom: var(--border);
+    }
+
+    .framework_ContentPlacement:nth-of-type(2),
+    .framework_ContentPlacement:nth-of-type(3),
+    .framework_ContentPlacement:nth-of-type(5),
+    .framework_ContentPlacement:nth-of-type(6) {
+        border-left: var(--border);
+    }
+
+    @media screen and (max-width: 1000px) {
+        .framework_container {
+            grid-template-columns: repeat(2, 1fr);
+        }
+
+        .framework_ContentPlacement:nth-of-type(2),
+        .framework_ContentPlacement:nth-of-type(4),
+        .framework_ContentPlacement:nth-of-type(6) {
+            border: none;
+            border-left: var(--border);
+        }
+
+        .framework_ContentPlacement:nth-of-type(1),
+        .framework_ContentPlacement:nth-of-type(3),
+        .framework_ContentPlacement:nth-of-type(5) {
+            border: none;
+            border-right: var(--border);
+        }
+
+        .framework_ContentPlacement:nth-of-type(1),
+        .framework_ContentPlacement:nth-of-type(2),
+        .framework_ContentPlacement:nth-of-type(3),
+        .framework_ContentPlacement:nth-of-type(4) {
+            border-bottom: var(--border);
+        }
+
+        .framework_ContentPlacement:nth-of-type(3),
+        .framework_ContentPlacement:nth-of-type(4),
+        .framework_ContentPlacement:nth-of-type(5),
+        .framework_ContentPlacement:nth-of-type(6) {
+            border-top: var(--border);
+        }
+    }
+
+    @media screen and (max-width: 700px) {
+        .framework_container,
+        .community_container {
+            grid-template-columns: 1fr;
+        }
+
+        .framework_ContentPlacement:nth-of-type(1),
```

Comment:
You might be able to reduce this down a few lines by setting:

```
.framewrok_contentPlacement:nth-of-type(1n) { 
    border: none;
    border-bottom: calc(var(--outline-width) * 1px) solid var(--neutral-outline-rest);
}

.framework_ContentPlacement:last-of-type {
    border: none;
}

Reply:
Thank you so much for this suggestion. I have went through and changed all nth-of-type to use n for increased reusability.